core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java

```diff
@@ -485,10 +366,10 @@ private TableUpdateDetails unsafeAssignTableToWriterThread(int tudKeyIndex, Char
         int leastLoad = Integer.MAX_VALUE;
         int threadId = 0;
 
-        for (int n = 0; n < loadByWriterThread.length; n++) {
-            if (loadByWriterThread[n] < leastLoad) {
-                leastLoad = loadByWriterThread[n];
-                threadId = n;
+        for (int i = 0, n = loadByWriterThread.length; i < n; i++) {
+            if (loadByWriterThread[i] < leastLoad) {
```

Comment:
nit: might be a good idea to have some kind of overflow protection since `TableUpdateDetails.eventsProcessedSinceReshuffle` is never reset now. With 100k rows/second it would take only around 6 hours (21474 seconds) for a writer thread to overflow its counter. Once this happens, the busiest thread may be considered the least loaded one by this method. The simplest fix would be to use `long` instead of an `int` type for the counter.

Reply:
done in https://github.com/questdb/questdb/pull/1767/commits/7e4b5c59557604b1bc8314593c51e1f7f4793f56